model/type.go

```diff
@@ -62,3 +62,8 @@ func (t *Type) RandomTensor(shape ...int) tensor.Tensor {
 	}
 	return ref
 }
+
+// OnesTensor create a tensor has 1 in all elements.
+func (t *Type) OnesTensor(shape ...int) tensor.Tensor {
```

Comment:
should be called `Ones`... it already returns a `Tensor`, so no need to repeat

Reply:
Sure!